source/collections.txt

```diff
@@ -70,33 +94,35 @@ Drop a Collection
 
 .. _collection-details:
 
-View Collection Details
------------------------
+Collection Details
+------------------
 
-The Collection Detail view shows detailed information for a
-collection, including the documents the collection contains. To view
+The Collection Detail screen shows detailed information for a
+collection, including the documents the collection contains. To see
 collection details, either:
 
 - Click a :guilabel:`Collection Name` in the main
-  :guilabel:`Collections` view, or
+  :guilabel:`Collections` screen, or
 
-- Click a collection in the left-hand pane.
+- Click a collection in the left navigation.
 
 .. figure:: /images/compass/select-collection.png
   :figwidth: 750px
 
-|compass| provides the following collection information in the
-detailed view:
+After you select a collection, |compass-short| shows you that
+collection's :doc:`/documents` tab. |compass-short| provides the following
+collection information and functionality in the detailed view:
 
 - :doc:`/documents`
-- :doc:`/schema` *Not Available in Compass Community Edition*
+- :ref:`compass-agg-builder`
+- :doc:`Schema Analysis </schema>` *Not Available in Compass Community Edition*
 - :doc:`/query-plan`
 - :doc:`/indexes`
 - :doc:`/validation` *Not Available in Compass Community Edition*
 
 You can also import and export documents to and from the selected
-collection from this view. For more information on this process,
-see :doc:`Import and Export Data </import-export/>`.
+collection from the Collection Detail screen. For more information on
+this process, see :doc:`Import and Export Data </import-export/>`.
```

Comment:
This is fine for now but I feel like we could have a bit more structure to indicate what's info and what's functionality. Are there other things you can do from the **Collection Detail** screen that we could highlight as "functionality"?

Reply:
Agreed this blurb about import/export is a bit out of place. I'll try adding it as another member of this bulleted list.